pom.xml

```diff
@@ -0,0 +1,187 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.teragrep</groupId>
+    <artifactId>cfe_39</artifactId>
+    <packaging>jar</packaging>
+    <version>${revision}${sha1}${changelist}</version>
+    <name>cfe_39</name>
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.8</maven.compiler.source>
+        <maven.compiler.target>1.8</maven.compiler.target>
+        <java.version>1.8</java.version>
+        <revision>0.0.1</revision>
+        <changelist>-SNAPSHOT</changelist>
+        <sha1/>
+        <metrics.version>4.2.8</metrics.version>
+        <prometheus-simpleclient.version>0.16.0</prometheus-simpleclient.version>
+        <hadoop.version>3.3.6</hadoop.version>
+    </properties>
+    <build>
+        <plugins>
```

Comment:
add spotless and other plugins from the https://github.com/teragrep/project-archetype/